templates/withdraw/approved-request-listing.php

```diff
@@ -21,7 +21,7 @@
     <?php foreach ( $requests as $row ) { ?>
         <tr>
             <td><?php echo wp_kses_post( wc_price( $row->amount ) ); ?></td>
-            <td><?php echo esc_html( dokan_withdraw_get_method_title( $row->method ) ); ?></td>
+            <td><?php echo esc_html( dokan_withdraw_get_method_title( $row->method ) ); ?> <?php echo esc_html( render_withdraw_method_details( $row ) ); ?></td>
```

Comment:
change `dokan_withdraw_get_method_title( $row->method )` with `dokan_withdraw_get_method_title( $row->method, $row )` and remove `<?php echo esc_html( render_withdraw_method_details( $row ) ); ?>`